src/content/stackoverflow/finder.js

```diff
@@ -16,10 +17,14 @@ const validURL = (href) => {
 const urlParsers = {
   'npmjs.com': npm.urlParser,
   'npmjs.org': npm.urlParser,
-  // 'pypi.org': python.urlParser,
-  // 'pypi.python.org': python.urlParser,
+  'pypi.org': python.urlParser,
+  'pypi.python.org': python.urlParser,
+  'packages.python.org': python.urlParser,
+  'pythonhosted.org': python.urlParser,
```

Comment:
verify if official and it is always pointing to the package name

Reply:
I think it is official.

- `packages.python.org` redirecting to `pythonhosted.org`.
- [WhoIs](https://www.whois.com/whois/pythonhosted.org) points to *Python Software Foundation*.